src/pages/lib/auth/manageusers/q/platform/[platform].mdx

```diff
@@ -1,11 +1,452 @@
+    });
+  } catch (e) {
+    console.log(e)
+  }
+}
+```
+
+</Block>
+<Block name="JavaScript">
+
+```js
+import { Auth } from 'aws-amplify'
+
+async function handleSignUp() {
+  try {
+    await Auth.signUp({
+      username: 'jdoe',
+      password: 'mysecurerandompassword#123',
+      attributes: {
+        email: 'me@domain.com',
+        phone_number: '+12128601234', // E.164 number convention
+        given_name: 'Jane',
+        family_name: 'Doe',
+        nickname: 'Jane',
+      }
+    });
+  } catch (e) {
+    console.log(e)
+  }
+}
+```
+
+</Block>
+
+</BlockSwitcher>
+
+### Retrieve user attributes
+
+You can then retrieve user attributes for your users to read in their profile. This helps you personalize their frontend experience as well as control what they will see. Visible attributes will be limited to those you enabled during the `Specify read attributes` configured with the Auth category. Then you can use the `currentAuthenticatedUser` method to retrieve user attributes.
+
+```ts
+const user = await Auth.currentAuthenticatedUser();
+
+const { attributes } = user;
+```
+
+Note that if you need to retrieve the latest user attributes after they are changed, you will need to bypass the local cache like so:
+
+```ts
+const user = await Auth.currentAuthenticatedUser({
+  bypassCache: true
+});
+```
+The `bypassCache` parameter defaults to false. Alternatively, your users can sign out and sign back in to retrieve and view their latest attributes.
+
+## Enable users to update, verify, and delete specific attributes
+
+You can enable your users to update, verify, and delete specific user attributes as their information changes over time. These attributes are enabled as writable when you update the `Specify write attributes` in your `Auth` configuration.
+
+### Update user attributes
+
+You can enable users to make updates after sign-up by using the `updateUserAttributes` method:
+
+<BlockSwitcher>
+<Block name="TypeScript">
+
+```ts
+import { Auth } from 'aws-amplify';
+
+async function updateUserAttributes () {
+  try {
+    const user = await Auth.currentAuthenticatedUser();
+    const result = await Auth.updateUserAttributes(user, {
+      email: 'me@anotherdomain.com',
+      family_name: 'Lastname'
+    });
+    console.log(result); // SUCCESS
+  } catch(err) {
+    console.log(err);
+  }
+};
+```
+
+</Block>
+<Block name="JavaScript">
+
+```js
+import { Auth } from 'aws-amplify';
+
+async function updateUserAttributes () {
+  try {
+    const user = await Auth.currentAuthenticatedUser();
+    const result = await Auth.updateUserAttributes(user, {
+      email: 'me@anotherdomain.com',
+      family_name: 'Lastname'
+    });
+    console.log(result); // SUCCESS
+  } catch(err) {
+    console.log(err);
+  }
+};
+```
+
+</Block>
+</BlockSwitcher>
+
+### Verify user attributes
+
+If your user changes an attribute that requires confirmation (for example, email or phone number), you will receive a confirmation code for that attribute. 
```

Comment:
When trying to update a user attribute that requires verification like email / phone_number and a code is sent, If the code is lost and the user wants to send the code again a second time they can invoke `Auth.updateUserAttributes` again to resend the code. (the latest code sent should be used for verification)

We could highlight this as well

Reply:
Fixing in v6 so leaving out for now